src/lib/OpenSkyApi.ts

```diff
@@ -42,6 +45,28 @@ export class OpenSkyApi {
     this._axios = axios.create(axiosConfig)
   }
 
+  public getFlightsByAircraft(
+    icao24: string | null,
+    beginTime: number | null,
+    endTime: number | null,
+  ){
```

Comment:
According to the docs [here](https://openskynetwork.github.io/opensky-api/rest.html#id11) the params `icao24`, `begin` and `end` are all required.

We should probably reflect that in the function signature.

```suggestion
  public getFlightsByAircraft(
    icao24: string,
    beginTime: number,
    endTime: number,
  ){
```